Apps/WebClient/test/functional/e2e/cypress/support/commands.js

```diff
@@ -35,4 +35,14 @@ Cypress.Commands.add("login", (username, password, authMethod = AuthMethod.BCSC)
         cy.get('#kc-login').click()
     }
     cy.get('#subject').should('have.text', 'Health Care Timeline')
-})
\ No newline at end of file
+});
+
+Cypress.Commands.add("checkTimelineHasLoaded", () => 
+{
+    cy.get('.spinner').should('not.exist');
```

Comment:
per our discussion today, can we find a way to use testid in a container of this spinner here?